packages/nextjs/pages/products/index.tsx

```diff
@@ -29,6 +29,9 @@ import { H6 } from "components/Typography/H6";
 import { Pagination } from "components/Pagination";
 import { FadeInOut } from "components/FadeInOut";
 import { Breadcrumbs } from "components/Breadcrumbs/Breadcrumbs";
+import { useDeviceSize } from "utils/useDeviceSize";
```

Comment:
organizational nit: put this utils import up with the other ones

Reply:
Done